.github/workflows/ensure-metadata-tests.yaml

```diff
@@ -0,0 +1,29 @@
+# Ensure lint metadata is tested
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+name: ensure-metadata-tests
+
+jobs:
+  ensure-metadata-tests:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: "release"
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+      - name: Ensure lint metadata is tested
```

Comment:
```suggestion
      - name: Ensure lint metadata is tested
      
```

Reply:
sorry, what's this for?

Reply:
Ah, crap. Sorry, the GitHub mobile UI caused this. 

I actually wanted there to be a space between `setup-r-dependencies` step and `Ensure lint metadata is tested` step to distinguish them.

Reply:
got it, done